File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/http/HttpSseEvent.java
package ru.iqchannels.sdk.http;


import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

class HttpSseEvent {
    @NonNull public final String id;
    @NonNull public final String name;
    @NonNull public final String data;

    HttpSseEvent(@Nullable String id, @Nullable String name, @Nullable String data) {
        this.id = id != null ? id : "";
        this.name = name != null ? name : "";
        this.data = data != null ? data : "";
    }
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/http/HttpSseEvent.kt
package ru.iqchannels.sdk.http

internal class HttpSseEvent(id: String?, name: String?, data: String?) {
	val id: String
	val name: String
	val data: String

	init {
		this.id = id ?: ""
		this.name = name ?: ""
		this.data = data ?: ""
	}
}
